compiler/plugins/input/Torch/InputConversion/Passes.cpp

```diff
@@ -49,6 +49,7 @@ void createTorchToIREEPipeline(
       mlir::torch::TorchConversion::createConvertCustomQuantOpPass());
   pm.addNestedPass<func::FuncOp>(
       torch::Torch::createDecomposeComplexOpsPass(emptyArrayRef));
+  pm.addNestedPass<func::FuncOp>(torch::Torch::createFuseQuantizedOpsPass());
```

Comment:
We are missing shape and dtype inference passes from the IREE pipeline, but I'm not sure if this is good to add or not. 

These shape/dtype inference passes are present in the torch-mlir pass pipeline `torch-lower-to-backend-contract` (which also applies `decompose-complex-ops`). So it might be reasonable to replace `torch-decompose-complex-ops` in this pipeline with `torch-lower-to-backend-contract` to clean up any decompositions that are written without explicit return types.

Reply:
Also, scalarize shapes would be useful to add. It might be useful to do that here.

Reply:
Any of those options/ideas sounds good to me. How do you want to proceed?

If there are particular public models failing before/after this, we can also add them to https://github.com/nod-ai/SHARK-TestSuite/tree/main/iree_tests. I've been watching efforts to test models in https://github.com/nod-ai/SHARK-TestSuite/tree/main/e2eshark and want to get more of those covered on presubmit here. Just need to get the test cases conforming to the right structure.

Reply:
I think it makes the most sense to rename and add scalarize shapes pass here for now. 

Some models might fail now because there is a memory allocation issue for several quantized models. E.g. the RAFT int8 model in the test-suite. I agree that we should add some int8 quantized public models to the iree_tests.